crates/client/assets/schema.sdl

```diff
@@ -1099,7 +1099,11 @@ type Query {
 		"""
 		The excluded coins from the selection.
 		"""
-		excludedIds: ExcludeInput
+		excludedIds: ExcludeInput,
+		"""
+		If true, returns available coins instead of failing when the requested amount is unavailable.
+		"""
+		allowPartial: Boolean! = false
```

Comment:
I think it should be optional and part of the `SpendQueryElementInput`. In this case user can specify which one can be filled partially, and which one can't.

Reply:
Addressed in https://github.com/FuelLabs/fuel-core/pull/2912